### ISSUE TITLE
Failed app start with: `chown: changing ownership of '/app/supervisord.conf': Operation not permitted`

### ISSUE THREAD

**Issue by calvingiles** [statianzo/dokku-supervisord#24]
```
2015-01-14 17:08:34,424 INFO supervisord started with pid 12
2015-01-14 17:08:35,427 INFO spawned: 'web' with pid 15
2015-01-14 17:08:35,477 DEBG 'web' stderr output:
chown: changing ownership of '/app/supervisord.conf': Operation not permitted

2015-01-14 17:08:35,478 DEBG fd 8 closed, stopped monitoring <POutputDispatcher at 140487649114304 for <Subprocess at 140487649113728 with name web in state STARTING> (stderr)>
2015-01-14 17:08:35,478 DEBG fd 6 closed, stopped monitoring <POutputDispatcher at 140487649113872 for <Subprocess at 140487649113728 with name web in state STARTING> (stdout)>
2015-01-14 17:08:35,478 INFO exited: web (exit status 1; not expected)
2015-01-14 17:08:35,478 DEBG received SIGCLD indicating a child quit
2015-01-14 17:08:36,480 INFO spawned: 'web' with pid 18
```

**Comment by kenips** [comment]
Hey @calvingiles mind trying https://github.com/kenips/dokku-supervisord/tree/bug/wrong_exec?

**Comment by calvingiles** [comment]
Works for me. :-)

**Comment by kenips** [comment]
Great! I'll get the PR in once I hear back from the others with #21 then.

**Comment by calvingiles** [comment]
Great - thanks

**Comment by kenips** [comment]
Fixed by #26
